### PR TITLE
Dev oukan

### PR DESCRIPTION
**Pull request by oxu971119**
It is comment.js from Oukan Xu.

### CODE REVIEW

File: data/comments.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    recipeThatComment.comments.push(newComment);
+    
+    //change String Id in comments into Object Id
+    let arr = recipeThatComment.comments;
+    
+
+    for(let i = 0; i < arr.length; i++){
+        for(let j in arr[i]){
+            if(j == '_id'){
+                arr[i][j] = ObjectId(arr[i][j]);
```
Comment (mihir-mj) [inline data/comments.js:33]: Why do we need to convert string into Object ID?

**Review by mihir-mj — COMMENTED** [review]
Check comment

File: data/comments.js (hunk 1 above)
Comment (oxu971119) [reply data/comments.js:33]: oh， i just copy it from lab6 , never mind

File: data/comments.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
+const mongoCollections = require('../config/mongoCollections')
+const recipes = require('./recipes');
+let { ObjectId } = require('mongodb');
+const recipesCo =  mongoCollections.recipes;
+const checkFunction = require('./verify');
```
Comment (mihir-mj) [inline data/comments.js:5]: Can you push the verify file??

**Pull request merged**